src/Toolbox.php

```diff
@@ -2622,16 +2622,20 @@ public static function in_array_recursive($needle, $haystack, $strict = false)
     /**
      * Slugify
      *
-     * @param string $string String to slugify
+     * @param string $str String to slugify
      * @param string $prefix Prefix to use (anchors cannot begin with a number)
      *
      * @return string
      */
-    public static function slugify($string, $prefix = 'slug_')
+    public static function slugify($str, $prefix = 'slug_')
```

Comment:
You should probably add a `bool $allow_special_chars = false` parameter to preserve previous behaviour by default. Indeed, special chars may be an issue when slug is used for an HTML id attribute or an URL.